### PR TITLE
include playground files into dist_build script

### PR DESCRIPTION
**Pull request by spetrunin**
Include playground assets into the old cloud build

## Related Issue
https://tyktech.atlassian.net/browse/TT-3784

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Refactoring or add test (improvements in base code or adds test coverage to functionality)

## Checklist

- [ ] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side). If pulling from your own
      fork, don't request your `master`!
- [ ] Make sure you are making a pull request against the **`master` branch** (left side). Also, you should start
      *your branch* off *our latest `master`*.
- [ ] My change requires a change to the documentation.
  - [ ] If you've changed APIs, describe what needs to be updated in the documentation.
  - [ ] If new config option added, ensure that it can be set via ENV variable
- [ ] I have updated the documentation accordingly.
- [ ] Modules and vendor dependencies have been updated; run `go mod tidy && go mod vendor`
- [ ] When updating library version must provide reason/explanation for this update.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] Check your code additions will not fail linting checks:
  - [ ] `go fmt -s`
  - [ ] `go vet`

### CODE REVIEW

File: bin/dist_build.sh [hunk 1]
```diff
@@ -71,13 +71,15 @@ mkdir -p $TEMPLATEDIR/middleware/python
...
-mkdir -p $TEMPLATEDIR/templates
+mkdir -p $TEMPLATEDIR/templates/playground
 mkdir -p $TEMPLATEDIR/policies
 mkdir -p $TEMPLATEDIR/utils
 mkdir -p $TEMPLATEDIR/install
 
 cp $SOURCEBINPATH/apps/app_sample.json $TEMPLATEDIR/apps
 cp $SOURCEBINPATH/templates/*.json $TEMPLATEDIR/templates
+cp $SOURCEBINPATH/templates/playground/index.html $TEMPLATEDIR/templates/playground
+cp $SOURCEBINPATH/templates/playground/playground.js $TEMPLATEDIR/templates/playground
```
Comment (excieve) [inline bin/dist_build.sh:82]: Perhaps future-proof it with this?
```suggestion
cp -R $SOURCEBINPATH/templates/playground/* $TEMPLATEDIR/templates/playground
```

**Review by excieve — APPROVED** [review]
Looks good! Perhaps make the copy more greedy to avoid changing this next time another template is added though?

**Review by alephnull — APPROVED** [review]

**Pull request merged**